### PR TITLE
Documentation for AutoBuild and AutoDeploy

### PR DESCRIPTION
**Pull request by ajesh-cc**
Hi Bart, Noah,

Updated KintoBlock Build Page to include Auto Build with a screenshot. We did not have this covered earlier. Also added content for Auto Deployment.

Please review and approve the changes.

Regards,
Ajesh

### CODE REVIEW

File: docs/kintoblock-builds.md [hunk 1]
```diff
@@ -47,8 +47,25 @@ Newly added or created branch in KintoBlock will be displayed as an individual c
...
 When changes are committed into the repository branch, a new build for KintoBlock can be triggered by using the **Build Latest Commit** button in the **KintoBlock Manage** page. Under **Recent Builds** user can view the recent commits made in the repository branch. The **Latest Build** section will be updated automatically with the last successful build.
 
-The **Automatically build new commits in this branch** option automatically enables build generation for new commits in the repository. By default, the toggle switch is set to `ON`.
+The **Automatically build new commits in this branch** option automatically enables build generation for new commits in the repository. By default, the toggle switch is set to `OFF`.
 
+## Automatically Build New Commits
+
+**Auto Build** toggle helps you to Automatically build your KintoBlock when there is a commit made in your Repository.
+
+ To enable `Auto Build`, follow the steps mentioned below:
```
Comment (bgever) [inline docs/kintoblock-builds.md]: No need to highlight as code

File: docs/kintoblock-builds.md [hunk 2]
```diff
@@ -47,8 +47,25 @@ Newly added or created branch in KintoBlock will be displayed as an individual c
...
+ 
+1. Create a new KintoBlock.
+
+2. Scroll down to the Advanced Option in Build & Repository.
+
+3. Enable Auto Build toggle
+
+4. Save the changes made.
+
+5. Now whenever you commit any changes to your repository, then your KintoBlock will be built automatically.
```
Comment (bgever) [inline docs/kintoblock-builds.md]: This is not a step, thus could be set without `5.`

File: docs/manage-deployments.md [hunk 3]
```diff
@@ -24,6 +24,22 @@ To delete a KintoBlocks, navigate to the KintoBlock section.
 2. Click on the **Deploy** button to apply the new changes to the deployments.
 
    ![Screenshot](/docs/assets/delete-kb-deployments.png)
+   
+## Automatically Deploy your Apps
+
+**Auto Deployment** toggle is used to automatically deploy and test light changes made in your successive builds of a KintoBlock version, without going through the hassle of manual deployment for each.
+
+To enable `Auto Deployment`, follow the steps mentioned below:
```
Comment (bgever) [inline docs/manage-deployments.md]: Same here, no need to format as code

File: docs/kintoblock-builds.md [hunk 4]
```diff
@@ -47,8 +47,25 @@ Newly added or created branch in KintoBlock will be displayed as an individual c
...
+
+   ![Screenshot](/docs/assets/Auto-Build.png)
```
Comment (bgever) [inline docs/kintoblock-builds.md]: Please describe what is in the screen shot, instead of `Screenshot`

File: docs/manage-deployments.md [hunk 5]
```diff
@@ -24,6 +24,22 @@ To delete a KintoBlocks, navigate to the KintoBlock section.
...
+
+1. Open an existing Deployment.
+
+2. Scroll down to the KintoBlocks section.
+
+3. Now enable Auto Deployment toggle.
+
+4. Currently, to save the changes made, we need to rebuild and deploy the KintoBlock or edit environment variables.
+
+   ![Screenshot](/docs/assets/Auto-Deployment.png)
```
Comment (bgever) [inline docs/manage-deployments.md]: Please describe what is in the screen shot, instead of `Screenshot`

File: docs/kintoblock-builds.md (hunk 1 above)
Comment (ajesh-cc) [reply docs/kintoblock-builds.md]: Done

File: docs/kintoblock-builds.md (hunk 2 above)
Comment (ajesh-cc) [reply docs/kintoblock-builds.md]: Done

File: docs/manage-deployments.md (hunk 3 above)
Comment (ajesh-cc) [reply docs/manage-deployments.md]: Done

File: docs/kintoblock-builds.md (hunk 4 above)
Comment (ajesh-cc) [reply docs/kintoblock-builds.md]: Done

File: docs/manage-deployments.md (hunk 5 above)
Comment (ajesh-cc) [reply docs/manage-deployments.md]: Done

**Pull request merged**